src/sentry/integrations/slack/tasks.py

```diff
@@ -170,6 +180,33 @@ def find_channel_id_for_alert_rule(organization_id, uuid, data, alert_rule_id=No
             redis_rule_status.set_value("failed")
             return
 
+    # This function call will write "input_channel_id" to the appropriate actions before we create the serializer
+    # I wasn't sure of a way to use the same loop to return True in the endpoints and to also do the lookup and didn't want to duplicate the loop
+    try:
+        mapped_ids = get_slack_channel_ids(organization, user, data)
+    except (serializers.ValidationError, ChannelLookupTimeoutError, InvalidTriggerActionError):
+        # channel doesn't exist error or validation error
+        logger.info(
+            "get_slack_channel_ids.failed",
```

Comment:
@iProgramStuff we should definitely log which error this was as well (ex: `error__class__.__name__`).

Reply:
Gonna log the exception. That way we also have the message as well and not just the class name.